packages/fluentui/docs/src/prototypes/table/ResponsiveTableContainer.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { TableCell, TableProps, Table } from '@fluentui/react-northstar';
+import calculateBreakpoints, { Column, Breakpoint } from './calculateBreakpoints';
+import * as _ from 'lodash';
+
+interface ResponsiveTableProps extends TableProps {
+  columns: Column[];
+  children: React.ReactElement<typeof Table>;
+  breakpoints?: number[];
+  id?: string;
+}
+
+const ResponsiveTableContainer: React.FC<ResponsiveTableProps> = props => {
+  const { columns: config, children } = props;
+
+  const responsiveTableID = React.useMemo(() => props.id || _.uniqueId(`${Table.className}__responsive-wrapper`), [
+    props.id,
+  ]);
+
+  const [breakpoints, setBreakpoints] = React.useState<Breakpoint>();
```

Comment:
```suggestion
  const [breakpoints, setBreakpoints] = React.useState<Breakpoint | undefined>();
```

As you don't have default value for `useState()`

Reply:
Done